cpp/CMakeLists.txt

```diff
@@ -642,7 +666,8 @@ add_library("${CUDF_PROXY_NAME}::${CUDF_PROXY_NAME}" ALIAS "${CUDF_PROXY_NAME}")
 ###################################################################################################
 # - link libraries --------------------------------------------------------------------------------
 
-target_link_libraries("${CUDF_PROXY_NAME}" "${CUDF_NAMESPACE}::${CUDF_BASE_NAME}" "${CUDF_MODULES}")
+target_link_libraries("${CUDF_PROXY_NAME}" "${CUDF_NAMESPACE}::${CUDF_BASE_NAME}" "${CUDF_MODULES}" 
+                        ZLIB::ZLIB Boost::filesystem)
```

Comment:
Should add these here, they should be added as link libraries against the underlying `CUDF_MODULES` instead of the top level proxy module.